hrms/public/js/expense_button.js

```diff
@@ -0,0 +1,32 @@
+function add_submit_button(frm){
+
+    if(frm.doc.status==="Draft"){
+        button_name="Send For Approval"
+    }else{
+        button_name="Approve"
+    }
+    
+    var method = null;
+    if(frm.doc.doctype==="Employee Advance"){
+        method="hrms.overrides.employee_advance.next_state"
+    } else if(frm.doc.doctype==="Expense Claim"){
+        method="hrms.overrides.expense_claim.next_state"
+    } else {
+        frappe.throw(_("Unhandled doctype"));
+    }
+
+    frm.add_custom_button(__(button_name), function() {
+        frappe.call({
+            method: method,
+            args: {
+                doc_name: frm.doc.name,
+            },
+            callback: function(r) {
+                if (r.message) {
+                    frm.refresh()
```

Comment:
rm frm.refresh

reload_doc will refresh it

Reply:
done